### ISSUE TITLE
sendnotification does not show app name on windows and never show icon you set

### ISSUE THREAD

**Issue by madaxemad** [fyne-io/fyne#1940]
hey
for notification whatever i do name of app don't change like changing app name or set it on compile and ....
but it show app name appID still on windows

**Comment by Jacalz** [comment]
We prefer if people use the [bug report / feature request issue templates](https://github.com/fyne-io/fyne/issues/new/choose) to make the issues more clear and understandable for us. Can you please explain more in depth at what issue you are seeing and what you would expect to see?

**Comment by madaxemad** [comment]
> if people use the [bug report / feature request issue templates](https://github.com/fyne-io/fyne/issues/new/choose) to make the issues more clear and understandable for us. Can you please explain more in depth at what issue you are seeing and what you would expect to see?

hey ok i really say it too ez after it for each 

for sending an notification must use
`app.SendNotification(&fyne.Notification{Title: "Title", Content: "Content"})`
it really have some rude problem it show name of app that sends notification appID on windows 
and 
also never show icon of app if set it before too 
it's not nice tbh

**Comment by andydotxyz** [comment]
Can you please share a screenshot of the notification and point out where the app id is displayed on Windows?
The notification should be just title and content text.

For android maybe this is a duplicate of #936.
On Windows this could be a feature request for "Show app icon in notification"?

**Comment by MatheusAlvesA** [comment]
I'm having the same problem.

Here is my code:
```go
package main

import (
	"time"
	"fyne.io/fyne/v2"
	"fyne.io/fyne/v2/app"
)

func main() {
	a := app.NewWithID("com.example.app")
	w := a.NewWindow("Test App")
	go showNotification(a)
	w.ShowAndRun()
}

func showNotification(a fyne.App) {
	time.Sleep(time.Second * 2)
	a.SendNotification(fyne.NewNotification("Example Title", "Example Content"))
}

```

**And here is the result:**
![image](https://user-images.githubusercontent.com/18595515/107860637-0ca34400-6e1f-11eb-96e9-0cb53be553a3.png)

**The expected result should be:**
![image](https://user-images.githubusercontent.com/18595515/107860787-2729ed00-6e20-11eb-9164-60fbccb1102e.png)

**Comment by stuartmscott** [comment]
app/app_windows.go#35
```
[Windows.UI.Notifications.ToastNotificationManager]::CreateToastNotifier("appID").Show($toast);`
```

**Comment by andydotxyz** [comment]
I have fixed the "appID" static text bug for 2.0.1 but I think we need a new `App.SetName(string)` for Aberlour to fix this correctly.

**Comment by Jacalz** [comment]
Can’t we just have it take the name of the first window instead? That should give us a better name than the appID, I think 🤔

**Comment by andydotxyz** [comment]
That crosses my mind, but it could also be irrelevant or empty (especially if a splash screen or tool window for example. Maybe that is a better fallback if no Id is set?

**Comment by Jacalz** [comment]
Well, I think the name of the first window would be better. From what I can see, most people use that as their application name and I would much prefer to have it say `wormhole-gui` than `com.github.jacalz.wormhole-gui`. If there is no name for it, we can either fall back to appID or just display “Fyne App”.

**Comment by andydotxyz** [comment]
I'm not sure this should have been closed @madaxemad can you say why you thought it was done?

**Comment by andydotxyz** [comment]
The name is fixed, the icon is also reported in #2592, closing this ticket and leaving the icon specific one open
